src/api/extension/request.ts

```diff
@@ -9,15 +9,18 @@ import {
     RequestArguments,
     RequestMethodEnum,
 } from '~/types/extension'
-import { apiEndpointSets } from '~/utils/constants'
+import { StorageEnum } from '~/types/storage'
+import { apiEndpointSets, chains } from '~/utils/constants'
 import { getStorage, setStorage } from '~/utils/extension/storage'
+import { puzzleHashToAddress } from '~/utils/signature'
 
 import * as Errors from './errors'
 import { permission } from './permission'
 
-const chainId = async (): Promise<string> => {
-    const chainId: string = await getStorage<string>('chainId')
-    return chainId || ChainEnum.Mainnet
+const chainId = async (): Promise<string | Errors.Error> => {
+    const chainId = await getStorage<string>(StorageEnum.chainId)
+
+    return chainId || Errors.InvalidParamsError
```

Comment:
I think we shouldn't return error here, if you want to return error should use `throw` keyword

Reply:
because use Errors message dapp can get error,bug use `throw` here just in background script